aten/src/ATen/native/Copy.cpp

```diff
@@ -115,13 +117,14 @@ void _copy_same_type__cpu(Tensor& self, const Tensor& src) {
     } else {
 #ifdef _OPENMP
```

Comment:
I think you can get rid of this. at::parallel does this as well. That means you can also get rid of the extra code below.